feat: `connectUrl()` instance method

charlypoly opened the pull request:
Enable a shorter pattern as follows:
```ts
import { Browserbase } from '@browserbasehq/sdk'
import { chromium } from "playwright-core";

(async () => {
  const browserbase = new Browserbase()
  const session = await browserbase.createSession({ fingerprint: { devices: ['mobile'] } })

  const browser = await chromium.connectOverCDP(
    browserbase.connectUrl({ sessionId: session.id, proxy: true })
  );

  //Getting the default context to ensure the sessions are recorded.
  const defaultContext = browser.contexts()[0];
  const page = defaultContext.pages()[0];

  await page.goto("https://browserbase.com/");
  await page.close();
  await browser.close();
})().catch((error) => console.error(error.message));
```

charlypoly committed:
feat: `connectUrl()` instance method

Enable a shorter pattern as follows:
```ts
import { Browserbase } from '@browserbasehq/sdk'
import { chromium } from "playwright-core";

(async () => {
  const browserbase = new Browserbase()
  const session = await browserbase.createSession({ fingerprint: { devices: ['mobile'] } })

  const browser = await chromium.connectOverCDP(
    browserbase.connectUrl({ sessionId: session.id, proxy: true })
  );

  //Getting the default context to ensure the sessions are recorded.
  const defaultContext = browser.contexts()[0];
  const page = defaultContext.pages()[0];

  await page.goto("https://browserbase.com/");
  await page.close();
  await browser.close();
})().catch((error) => console.error(error.message));
```

mishushakov commented:
what about the `load` and `loadURLs` methods, can we re-use connectUrl there too?

mishushakov commented:
on casing, why Url instead of URL?

charlypoly commented:
> on casing, why Url instead of URL?

@mishushakov I followed the convention used by our Sessions API, for example here with `debuggerFullscreenUrl`: https://docs.browserbase.com/api-reference/retrieve-a-session-debug-connection-urls

charlypoly committed:
refactor: have `loadURL()` and `loadURLs()` share `ConnectOptions`

charlypoly commented:
> da34bc7

Fixed in [da34bc7](https://github.com/browserbase/js-sdk/pull/3/commits/da34bc73aa5e917d56b5561ee99867a387aa8648), I also refactored `LoadOptions` with `ConnectOptions`
